apps/member/src/hooks/queries/activity/useActivityGroupMemberMutation.ts

```diff
@@ -13,23 +14,27 @@ export function useActivityGroupMemberMutation() {
 
   const mutation = useMutation({
```

Comment:
`mutation` 불러와서 사용하는게 `mutate`, `isPending` 두가지라면, 간단하게 위에서 정리해보는것도 좋아보입니다.
꼭 아래와 같이 할 필요는 없어용

```suggestion
  const { mutate: activityGroupMemberMutate, isPending: activityGroupMemberIsPending } = useMutation({
```
<br/>
위처럼 작성하면 아래에서 그냥 간단하게 `return` 해줄 수 있겠네요

```ts
  return { activityGroupMemberMutate, activityGroupMemberIsPending };
```

Reply:
[974f843](https://github.com/kgu-clab/clab-platforms/pull/221/commits/974f8434fb4096f0d43176c1d3adec564d536dff) 더 알아보기 쉬운 방법인 것 같아서 수정하였습니다! 감사해요